### ISSUE TITLE
Beautiful!

### ISSUE THREAD

**Issue by nadbm** [harrisrobin/cows-bulls-chatbot#1]
Hey @harrisrobin , I've been working on something like this. Nowhere near pretty as this.
You should make this into a npm package, similar to 
https://github.com/FaridSafi/react-native-gifted-chat

Any plans for that?

**Comment by harrisrobin** [comment]
@nadbm funny you mention that, i've been looking for the exact same thing and could not find anything that's good enough. I'd love to work on that. I won't have time this month but let's discuss it and perhaps we can collaborate :)

**Comment by nadbm** [comment]
Awesome, let's do it. Ideally I would love to have a similar api to the gifted-chat. I might have some time later next week, I can get it started by forking this repo!

**Comment by harrisrobin** [comment]
@nadbm i agree about keeping the API consistent with gifted-chat. We might be better off starting a new repo, the original purpose of this one was really just a demo/proof-of-concept of something. 

I'll be in touch later this week ;)
